File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/inventory/DrawableInventory.java
package com.gmail.borlandlp.minigamesdtools.gui.inventory;

import com.gmail.borlandlp.minigamesdtools.gui.inventory.slots.InventorySlot;

public interface DrawableInventory {
    org.bukkit.inventory.Inventory toBukkitInventory();
    InventorySlot getSlot(int id);
    void setSlot(int id, InventorySlot slot);
    void setRows(int s);
    int getRows();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/inventory/DrawableInventory.kt
package com.gmail.borlandlp.minigamesdtools.gui.inventory

import com.gmail.borlandlp.minigamesdtools.gui.inventory.slots.InventorySlot
import org.bukkit.inventory.Inventory

interface DrawableInventory {
    fun toBukkitInventory(): Inventory
    fun getSlot(id: Int): InventorySlot
    fun setSlot(id: Int, slot: InventorySlot)
    var rows: Int
}